app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -232,13 +237,30 @@ public void onImageProcessed(UploadItem uploadItem, Place place) {
     }
 
     /**
-     * Shows popup if any nearby location needing pictures matches uploadable picture's GPS location
+     * Sets variables to Show popup if any nearby location needing pictures matches uploadable picture's GPS location
      * @param uploadItem
      * @param place
      */
-    @SuppressLint("StringFormatInvalid")
     @Override
     public void onNearbyPlaceFound(UploadItem uploadItem, Place place) {
+        nearbyPlace = place;
+        this.uploadItem = uploadItem;
+        showNearbyFound = true;
+        if(showNearbyFound && callback.getIndexInViewFlipper(this) == 0) {
```

Comment:
Is `showNearbyFound &&` really needed after the `showNearbyFound = true;` at the previous line?